draft-ietf-tls-tls13.md

```diff
@@ -1377,26 +1388,19 @@ those for other kinds of TLS data.  Specifically:
 keys derived using the offered PSK.
 
 2. There are no guarantees of non-replay between connections.
-Unless the server takes special measures outside those provided by TLS,
-the server has no guarantee that the same
-0-RTT data was not transmitted on multiple 0-RTT connections
-(see {{replay-time}} for more details).
-This is especially relevant if the data is authenticated either
-with TLS client authentication or inside the application layer
-protocol. However, 0-RTT data cannot be duplicated within a connection (i.e., the server
-will not process the same data twice for the same connection) and
-an attacker will not be able to make 0-RTT data appear to be
-1-RTT data (because it is protected with different keys.)
-
-Protocols MUST NOT use 0-RTT data without a profile that defines its
-use. That profile needs to identify which messages or interactions are
-safe to use with 0-RTT. In addition, to avoid accidental misuse,
-implementations SHOULD NOT enable 0-RTT unless specifically
-requested. Implementations SHOULD provide special functions for 0-RTT data to ensure
-that an application is always aware that it is sending or receiving
-data that might be replayed.
-
-The same warnings apply to any use of the early_exporter_master_secret.
+Anti-replay for ordinary TLS 1.3 1-RTT data is provided via the
```

Comment:
Is this intended to be one giant paragraph?  I think that it could probably be three.